keyboards/keebio/levinson/keymaps/ksamborski/keymap.c

```diff
@@ -0,0 +1,158 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _QW,
+  _LW,
+  _RS,
+  _AD,
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+  MKITPNK,
+  DYNAMIC_MACRO_RANGE
+};
+
+#define DREC_1  DYN_REC_START1
+#define DREC_2  DYN_REC_START2
+#define DPLAY_1 DYN_MACRO_PLAY1
+#define DPLAY_2 DYN_MACRO_PLAY2
+#define DSTOP   DYN_REC_STOP
```

Comment:
The core implementation also has short-form keycodes built-in:

https://github.com/qmk/qmk_firmware/blob/2c1417246789383829145bdfc51b3b3921a99341/quantum/quantum_keycodes.h#L796-L801

If you use the core shortcodes you can delete Lines-19-23.